src/crypto/rust_crypto/rsa.rs

```diff
@@ -0,0 +1,145 @@
+//! Implementations of the [`JwtSigner`] and [`JwtVerifier`] traits for the
+//! RSA family of algorithms using [`aws_lc_rs`]
```

Comment:
```suggestion
//! RSA family of algorithms using [`RustCrypto`]
```

Reply:
Ah, good catch, missed that. Pushed a fix.